godot_project/editor/rendering/atomic_structure_renderer/representation/stick_representation/stick_representation.gd

```diff
@@ -196,9 +198,6 @@ func lowlight_atoms(_in_atoms_ids: PackedInt32Array, in_bonds_released_from_part
 			new_partially_influenced_bonds: PackedInt32Array = PackedInt32Array()) -> void:
 	_refresh_bond_partial_influence_status(in_bonds_released_from_partial_influence)
 	_refresh_bond_partial_influence_status(new_partially_influenced_bonds)
-	if _current_bond_partial_selection.size() == in_bonds_released_from_partial_influence.size():
-		_current_bond_partial_selection.clear()
```

Comment:
this actually lead to third issue which I haven't documented, this assumed that since there is the same nmb of bonds that just have been released from partial influence as there have been in the past those two collections are the same. 
I had edge case where it's not true (for case where both collections have size = 1)

This check is totally unneeded after changes from 184-188 lines